### ISSUE TITLE
Error on build creation 

### ISSUE THREAD

**Issue by riccardolorenzon** [cloverfield-tools/universal-react-boilerplate#106]
I get this error when i run `npm run build:dev`. 

ERROR in ./source/client/index.js
Module build failed: SyntaxError: [...]/source/client/index.js: Unexpected token (15:4)
  13 | const store = createStore(
  14 |   combineReducers({
> 15 |     ...reducers,
     |     ^
  16 |     routing: routerReducer
  17 |   }),
  18 |   // hydrating server.
   
Any suggestions?

**Comment by ericelliott** [comment]
Yep. Looks like we don't have object spread -- could be the babel config or the .eslint config (probably the first). In fact, it looks like we've fallen behind on babel & the config that goes with it. Would you like to take a crack at a PR?
